config.json

```diff
@@ -8,6 +8,7 @@
     "hello-world",
     "anagram",
     "leap",
+    "gigasecond",
```

Comment:
In addition to the `problems` list (which has been deprecated), there's an `exercises` list below. Can you add this exercise to that list as well? See https://github.com/exercism/xc/issues/37 for more information.